<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="clientTab">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QLineEdit" name="inputIpAddress">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>201</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>192.168.1.10</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connectButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QLabel" name="infoLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="serverTab">
     <attribute name="title">
      <string>Server</string>
     </attribute>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>450</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="availableMonitors">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>211</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Available screens</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="identifyScreens">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Identify screens</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
